src/app/components/updatedList/updatedList.scss

```diff
@@ -0,0 +1,80 @@
+@import "styles/_base";
+@import "styles/_colors";
+
+$advancedsearchimagesize1: 66px;
+$advancedsearchimagesize2: 105px;
+$advancedsearchimagesize3: 116px;
+$advancedsearchimagesize_mobile: 85px;
+
```

Comment:
This css is going to leak to all the other html files aswell. Please look at the profile.scss file under /components/profile to see an example on how to wrap the scss in a correct way